package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "scripts": {
+    "ci:set-env": "zx ./scripts/ci/set-env.mjs",
+    "programs:build": "zx ./scripts/program/build.mjs",
+    "programs:test": "zx ./scripts/program/test.mjs",
+    "programs:clean": "zx ./scripts/program/clean.mjs",
+    "programs:format": "zx ./scripts/program/format.mjs",
+    "programs:lint": "zx ./scripts/program/lint.mjs",
```

Comment:
What do you think if we have a `programs:clippy` for running clippy only? Then the `programs:lint` can be used to invoke both:
```
"programs:lint": "pnpm programs:format && pnpm programs:clippy"
```

Reply:
Yes, and then use `programs:lint` in CI, right? Tell it to fail on warnings.

Reply:
As per [this comment here](https://github.com/solana-program/feature-gate/pull/8#discussion_r1661326060), I'm going to address formatting and linting in the next PR. These came straight from the template.